### DISCUSSION TITLE
Broadcast notifications doesn't popup

### DISCUSSION THREAD

**Discussion by nordiccharge** [filamentphp/filament#13403] [Help]
### Package

Panel builder

### Package Version

v3.2.82

### How can we help you?

Hi guys

I'm having a hard time getting broadcast notifications working with Pusher. I have tried with and without `composer require filament/notifications:"^3.0-stable" -W` even though it also states in the documentation that it is bundled with the Panel builder.

Pusher actually sends the messages but they never popup.

I have already read this issue here: https://github.com/filamentphp/filament/discussions/10163 but the solution doesn't work for me.

I'm running a job in queue with the following code:
`Notification::make()
       ->title("#{$id} : CREATED on Monta")
       ->body($response->body())
       ->icon('heroicon-o-check-circle')
       ->iconColor('success')
       ->sendToDatabase($user)
       ->broadcast($user);`

My filament config looks like this:
`    'broadcasting' => [

        'echo' => [
            'broadcaster' => 'pusher',
            'key' => env('VITE_PUSHER_APP_KEY'),
            'cluster' => env('VITE_PUSHER_APP_CLUSTER'),
            'wsHost' => env('VITE_PUSHER_HOST'),
            'wsPort' => env('VITE_PUSHER_PORT'),
            'wssPort' => env('VITE_PUSHER_PORT'),
            'authEndpoint' => '/api/v1/broadcasting/auth',
            'disableStats' => true,
            'encrypted' => true,
        ],

    ],`

Is there anything I am missing?

**Comment by Ticket-Master** [comment]
Hi, 
Did you get it working ? because i am facing the same issue, i have reverb running fine, and the jobs are being sent but in Notifications it is not showing it shows only after i refresh the page.

**Reply by nordiccharge** [reply]
Sadly, I did not. I have actually put the broadcast notifications for my project in hold for now.

**Reply by vermaysha** [reply]
same, reverb running fine and jobs are being sent, but no notification appear

**Reply by keetamhoang** [reply]
I have a same issue, anyone knows?

**Reply by xBigDaddyx** [reply]
same issue

**Comment by SuperUserDo-Nikola** [comment]
I had also trouble with it, but managed to make it work. This was my setup with reverb.

reverb.php
`<?php

return [

    /*
    |--------------------------------------------------------------------------
    | Default Reverb Server
    |--------------------------------------------------------------------------
    |
    | This option controls the default server used by Reverb to handle
    | incoming messages as well as broadcasting message to all your
    | connected clients. At this time only "reverb" is supported.
    |
    */

    'default' => env('REVERB_SERVER', 'reverb'),

    /*
    |--------------------------------------------------------------------------
    | Reverb Servers
    |--------------------------------------------------------------------------
    |
    | Here you may define details for each of the supported Reverb servers.
    | Each server has its own configuration options that are defined in
    | the array below. You should ensure all the options are present.
    |
    */

    'servers' => [

        'reverb' => [
            'host' => env('REVERB_SERVER_HOST', '0.0.0.0'),
            'port' => env('REVERB_SERVER_PORT', 8080),
            'hostname' => env('REVERB_HOST'),
            'options' => [
                'tls' => [],
            ],
            'max_request_size' => env('REVERB_MAX_REQUEST_SIZE', 10_000),
            'scaling' => [
                'enabled' => env('REVERB_SCALING_ENABLED', false),
                'channel' => env('REVERB_SCALING_CHANNEL', 'reverb'),
                'server' => [
                    'url' => env('REDIS_URL'),
                    'host' => env('REDIS_HOST', '127.0.0.1'),
                    'port' => env('REDIS_PORT', '6379'),
                    'username' => env('REDIS_USERNAME'),
                    'password' => env('REDIS_PASSWORD'),
                    'database' => env('REDIS_DB', '0'),
                ],
            ],
            'pulse_ingest_interval' => env('REVERB_PULSE_INGEST_INTERVAL', 15),
            'telescope_ingest_interval' => env('REVERB_TELESCOPE_INGEST_INTERVAL', 15),
        ],

    ],

    /*
    |--------------------------------------------------------------------------
    | Reverb Applications
    |--------------------------------------------------------------------------
    |
    | Here you may define how Reverb applications are managed. If you choose
    | to use the "config" provider, you may define an array of apps which
    | your server will support, including their connection credentials.
    |
    */

    'apps' => [

        'provider' => 'config',

        'apps' => [
            [
                'key' => env('REVERB_APP_KEY'),
                'secret' => env('REVERB_APP_SECRET'),
                'app_id' => env('REVERB_APP_ID'),
                'options' => [
                    'host' => env('REVERB_HOST'),
                    'port' => env('REVERB_PORT', 443),
                    'scheme' => env('REVERB_SCHEME', 'https'),
                    'useTLS' => env('REVERB_SCHEME', 'https') === 'https',
                ],
                'allowed_origins' => ['*'],
                'ping_interval' => env('REVERB_APP_PING_INTERVAL', 60),
                'max_message_size' => env('REVERB_APP_MAX_MESSAGE_SIZE', 10_000),
            ],
        ],

    ],

];
`
filament.php
`    'broadcasting' => [

         'echo' => [
             'broadcaster' => 'reverb',
             'key' => env('VITE_REVERB_APP_KEY'),
             'cluster' => env('VITE_REVERB_APP_CLUSTER'),
             'wsHost' => env('VITE_REVERB_HOST'),
             'wsPort' => env('VITE_REVERB_PORT') ?? 80,
             'wssPort' => env('VITE_REVERB_PORT') ?? 443,
             'authEndpoint' => '/broadcasting/auth',
             'disableStats' => true,
             'encrypted' => env('REVERB_SCHEME', 'https') === 'https',
             'forceTLS' => env('REVERB_SCHEME', 'https') === 'https',
         ],

    ],
`

.env 
`QUEUE_CONNECTION=database
REVERB_APP_ID=
REVERB_APP_KEY=
REVERB_APP_SECRET=
REVERB_HOST="ricambi.test" // Should be same as APP_URL without http://
REVERB_PORT=8080
REVERB_SCHEME=https // Based on your local or prod env

VITE_REVERB_APP_KEY="${REVERB_APP_KEY}"
VITE_REVERB_HOST="${REVERB_HOST}"
VITE_REVERB_PORT="${REVERB_PORT}"
VITE_REVERB_SCHEME="${REVERB_SCHEME}"`

echo.js 
`import Echo from 'laravel-echo';

import Pusher from 'pusher-js';
window.Pusher = Pusher;

window.Echo = new Echo({
    broadcaster: 'reverb',
    key: import.meta.env.VITE_REVERB_APP_KEY,
    wsHost: import.meta.env.VITE_REVERB_HOST,
    wsPort: import.meta.env.VITE_REVERB_PORT ?? 80,
    wssPort: import.meta.env.VITE_REVERB_PORT ?? 443,
    forceTLS: (import.meta.env.VITE_REVERB_SCHEME ?? 'https') === 'https',
    enabledTransports: ['ws', 'wss'],
});`

From commands i used
php artisan reverb:start
php artisan queue:work

And check what network tab under "ws" says
